src/theme.ts

```diff
@@ -1,33 +1,70 @@
-/* eslint-disable @mll-lab/no-color-literals */
-
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
 
+/* eslint-disable @mll-lab/no-color-literals */
 /**
- * We omit special purpose colors:
- * - 'white': Brightest possible
- * - 'black': Darkest possible
- * - 'transparent': No color
+ * Inspired by https://ant.design/docs/spec/colors
  */
-export const MLL_THEME = {
-  borderColor: '#50a0d0',
-  dividerColor: '#dadada',
-  tableBorderColor: '#e8e8e8',
-  containerBorderColor: '#d3d3d3',
-  backgroundColor: '#f0f2f5',
-  panelBackgroundColor: '#eaeaea',
-  collapseBackgroundColor: '#d4edfc',
-  focusedRowColor: '#d4edfc',
-  titleColor: '#c0c0c0',
-  menuGroupBackgroundColor: '#fafafa',
-  successColor: '#52c41a',
-  warningColor: '#faad14',
-  errorColor: '#f5222d',
-  infoColor: '#d9d9d9',
+export const PALETTE = {
+  red: '#ff0000', // matches the MLL corporate identity
+  gold: '#ffc53d',
+  green: '#bae637',
+  blue: '#50a0d0',
+  lightBlue: '#d4edfc',
+  blueTintedGray: '#f0f2f5',
+
+  // Shades
+  white: '#ffffff', // brightest possible
+  gray1: '#fafafa',
+  gray2: '#f0f0f0',
+  gray3: '#e6e6e6',
+  gray4: '#d9d9d9',
+  gray5: '#c0c0c0',
+  gray6: '#8c8c8c',
+  gray7: '#595959',
+  gray8: '#434343',
+  gray9: '#262626',
+  black: '#000000', // darkest possible
```

Comment:
@spawnia I've added a gray scale that should suit our needs and partly mapped the current hex values to it. 
The remaining deprecated values have been extracted to a new const and should be replaced in limes-frontend with the new values available

Reply:
nice!